modules/oadp-about-backing-and-restoring-from-cluster-to-cluster.adoc

```diff
@@ -0,0 +1,84 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+
+:_content-type: CONCEPT
+[id="oadp-about-backing-and-restoring-from-cluster-to-cluster_{context}"]
+= About backing up data from one cluster and restoring it on another cluster
+
+{oadp-first} is designed to back up and restore application data in the same {product-title} cluster. {mtc-full} ({mtc-short}) is designed to migrate containers, including application data, from one {product-title} cluster to another.
+
+You can use OADP to back up application data from one {product-title} cluster and restore it on another cluster. However, doing so is more complicated than using {mtc-short} or using OADP to back up and restore on the same cluster.
+
+To successfully use OADP to back up data from one cluster and restore it to another cluster, you must take into account the following factors, in addition to the prerequisites and procedures that apply to using OADP to back up and restore data on the same cluster:
+
+* Operators
+* Use of Velero
+* UID and GID ranges
+
+[id="oadp-cluster-to-cluster-operators_{context}"]
+== Operators
+You must exclude Operators from the backup of an application for backup and restore to succeed.
+
+[id="oadp-cluster-to-cluster-velero_{context}"]
+== Use of Velero
+
+Velero, which OADP is built upon, does not natively support migrating persistent volume snapshots across cloud providers. To migrate volume snapshot data between cloud platforms, you must _either_ enable the Velero Restic option `file-system-backup`, which backs up volume contents at the filesystem level, _or_ use the OADP Data Mover for CSI snapshots.
+
+[NOTE]
+====
+In OADP 1.2 and later, the Velero Restic option is called `file-system-backup`.
+====
+
+[NOTE]
+====
+Velero's file system backup feature supports both Kopia and Restic, but currently OADP supports only Restic.
+====
+
+For more information about using Restic with OADP, see link:https://access.redhat.com/documentation/en-us/openshift_container_platform/4.13/html-single/backup_and_restore/index#oadp-backing-up-applications-restic_backing-up-applications[backing up applications with Restic].
```

Comment:
Why are we linking to a specific version of the OCP docs in the portal? Is it appropriate for, say, 4.10 customers to use the 4.13 docs? Eventually, too, the link will age out. 
Why not use Additional references in the assembly?

Reply:
Too many `../` I think.

`xref:../../backup_and_restore/application_backup_and_restore/backing_up_and_restoring/backing-up-applications.adoc#oadp-backing-up-applications-restic_backing-up-applications[Backing up applications with Restic]`

Reply:
Got it!